web/templates/settings/settings_numeric_input.hbs

```diff
@@ -0,0 +1,18 @@
+{{#unless (eq render_only false)}}
+<div class="input-group setting-next-is-related {{#if is_disabled}}control-label-disabled{{/if}}">
```

Comment:
What does `setting-next-is-related` do? It seems to already exist, but also seems to not do anything.